src/InputHandler.js

```diff
@@ -0,0 +1,58 @@
+import { Console } from '@woowacourse/mission-utils';
+
+class InputHandler {
+    async getCarNames() {
+        const carNames = await Console.readLineAsync('경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n');
+        this.validateCarNames(carNames);
+        return this.splitCars(carNames);
+    }
+
+    validateCarNames(carNames) {
+        if (carNames === '') {
+            throw new Error('[ERROR] 경주 하려면 자동차가 2대 이상 필요해요!');
+        }
+
+        const names = carNames.split(',');
+        const hasEmptyName = names.some(name => name === '');
+        const nameTooLong = names.some(name => name.length > 5);
+        const notEnoughCars = names.length < 2;
+        const hasDuplicates = new Set(names).size !== names.length;
+
+        if (hasEmptyName) {
+            throw new Error('[ERROR] 누군가 참가용지에 이름을 쓰지 않았습니다!');
+        } else if (nameTooLong) {
+            throw new Error('[ERROR] 자동차 이름은 5자 이하로 입력해주세요!');
+        } else if (notEnoughCars) {
+            throw new Error('[ERROR] 경주 하려면 자동차가 2대 이상 필요해요!');
+        } else if (hasDuplicates) {
+            throw new Error('[ERROR] 중복된 자동차 이름이 있어요!');
+        }
```

Comment:
```suggestion
        if (hasEmptyName) {
            throw new Error('[ERROR] 누군가 참가용지에 이름을 쓰지 않았습니다!');
        } 
        if (nameTooLong) {
            throw new Error('[ERROR] 자동차 이름은 5자 이하로 입력해주세요!');
        } 
        if (notEnoughCars) {
          throw new Error('[ERROR] 경주 하려면 자동차가 2대 이상 필요해요!');
        } 
        if (hasDuplicates) {
            throw new Error('[ERROR] 중복된 자동차 이름이 있어요!');
        }
```

같은 형태로 나타내시거나
각각의 에러 컨디션들을 함수로 만들어 준 후
```suggestion
// other file ex) validator.js
const hasEmptyNames = (names) => names.some(name => name === EMPTY);
...
const validateRules = [hasEmptyName, nameTooLong, notEnoughCars, hasDuplicates];
validateRules.forEach(fn => fn(names));
```
이렇게 검증하는 건 어떨까요?:)

Reply:
데박